src/node.cc

```diff
@@ -303,6 +305,18 @@ MaybeLocal<Value> StartExecution(Environment* env, StartExecutionCallback cb) {
     first_argv = env->argv()[1];
   }
 
+#ifndef DISABLE_SINGLE_EXECUTABLE_APPLICATION
+  if (per_process::sea::IsSingleExecutable()) {
+    // TODO(addaleax): Find a way to reuse:
```

Comment:
Should this TODO be addaleax ?

Reply:
:woman_shrugging: I’m fine with it.

Reply:
yea, Anna originally suggested this in https://github.com/nodejs/node/pull/45038#discussion_r1090626101 and she probably has most of the context on how this could be done, so I thought putting her name here would be better